tasks/upgrade_gem.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Check if gem is installed
+  command: 'which gem'
+  register: which_gem
+  changed_when: False
+  failed_when: False
+
+- name: Update gems
+  command: 'gem update'
```

Comment:
why are we not using https://docs.ansible.com/ansible/latest/modules/gem_module.html

Reply:
Because if it is not installed on the target host, the task will fail, even if one uses a `when` condition that evaluates to false.

Reply:
actually if you include a file with a when condition dynamically this should not happend but ok